pygrocy/grocy.py

```diff
@@ -607,3 +607,6 @@ def recipe(self, recipe_id: int) -> RecipeItem:
         recipe = self._api_client.get_recipe(recipe_id)
         if recipe:
             return RecipeItem(recipe)
+
+    def add_generic(self, entity_type, data):
```

Comment:
this is fine as a first version, but we should consider converting `entity_type` to a dedicated enum